volttrontesting/fixtures/volttron_platform_fixtures.py

```diff
@@ -109,9 +110,9 @@ def cleanup():
     request.addfinalizer(cleanup)
     return wrapper
 
-
+# IPC testing is removed since it is not used from VOLTTRON 6.0
 @pytest.fixture(scope="function",
-        params=['tcp', 'ipc'])
+        params=['tcp'])
```

Comment:
If there is only one param we shouldn't really parametrize the test. Avoid the additional overload.

Reply:
done